tests/test_vrf.rb

```diff
@@ -84,11 +84,15 @@ def test_shutdown
     end
     v.shutdown = true
     assert(v.shutdown)
+
+    sleep 20 if node.product_id[/N(5|6)/]
```

Comment:
Yeah you'd have to try it, rescue CliError, sleep a few, try again until either it succeeds or you exceed a maximum retry count.